en/peripherals/mavlink_peripherals.md

```diff
@@ -39,13 +53,16 @@ The `TELEM 1` port is almost always used for the GCS telemetry stream.
 To support this there is a [default serial port mapping](../peripherals/serial_configuration.md#default_port_mapping) of MAVLink instance 0 as shown below:
 - [MAV_0_CONFIG](../advanced_config/parameter_reference.md#MAV_0_CONFIG) = `TELEM 1`
 - [MAV_0_MODE](../advanced_config/parameter_reference.md#MAV_0_MODE) = `Normal`
-- [MAV_0_RATE](../advanced_config/parameter_reference.md#MAV_0_RATE)= `57600` baud
+- [MAV_0_RATE](../advanced_config/parameter_reference.md#MAV_0_RATE)= `1200` B/s

```

Comment:
Note this is the default rate from the parameter docs. It is correct right?

Reply:
@bkueng So based on your example for the MAV_2_RATE this could be 4800 or similar - but I'll leave as 1200 as this is what the param docs says the default value of MAV_0_RATE is in param docs - unless you say otherwise.